packages/gafl-webapp-service/src/__tests__/server.telesales.spec.js

```diff
@@ -0,0 +1,17 @@
+import { start, stop, initialize, injectWithCookies } from '../__mocks__/test-utils.js'
+import { LICENCE_LENGTH } from '../uri.js'
+
+beforeAll(d => {
+  process.env.CHANNEL = 'telesales'
+  start(d)
+})
+
+beforeAll(d => initialize(d))
+afterAll(d => stop(d))
```

Comment:
Should we unset `process.env.CHANNEL` so that other tests aren't impacted?
```
const OLD_ENV = process.env
beforeAll(d => {
  process.env.CHANNEL = 'telesales'
  start(d)
  initialise(d)
})
afterAll(d => {
  stop(d)
  process.env = OLD_ENV
})

Reply:
I think each module starts a separate process but yes I will do that for good measure